worker/src/main/scala/org/apache/celeborn/service/deploy/worker/WorkerSource.scala

```diff
@@ -18,13 +18,15 @@
 package org.apache.celeborn.service.deploy.worker
 
 import org.apache.celeborn.common.CelebornConf
+import org.apache.celeborn.common.internal.Logging
 import org.apache.celeborn.common.metrics.MetricsSystem
 import org.apache.celeborn.common.metrics.source.AbstractSource
+import org.apache.celeborn.service.deploy.worker.WorkerSource._
 
-class WorkerSource(conf: CelebornConf) extends AbstractSource(conf, MetricsSystem.ROLE_WORKER) {
+class WorkerSource(conf: CelebornConf) extends AbstractSource(conf, MetricsSystem.ROLE_WORKER)
+  with Logging {
```

Comment:
Why with Logging? Should remove?

Reply:
@AngersZhuuuu, the invoker of `startCleaner()` in `WorkerSource` needs with logging.

Reply:
Actually. `startCleaner()` will use log from `Utils`. This is not necessary.